workspace/welcome-to-xod/012-strings/patch.xodp

```diff
@@ -1,15 +1,29 @@
 {
   "comments": [
     {
-      "content": "# Strings\n\nAnother important data type is *strings*. Strings represent pieces of text like “Hello World!” or \"[https://xod.io](https://xod.io/?utm_source=ide&utm_medium=ide_comment&utm_campaign=tutorial_012)\".\n\n## Instructions\n\n1. Run the Simulation.\n2. Observe how the message alternates between \"Hello World\" and \"Hello XOD\".\n3. Change values of `tweak-string-16`s to change the message.\n4. Add a third pin to `concat` and bind an exclamation point to it.\n\nNotice that tweak nodes for strings are a bit special. For optimisation purposes XOD needs to know the maximum length of the tweaked string, so there are several ones available. Try to choose the smallest ones.",
+      "content": "## 👆 Generics\n\nNote, the `if-else` node can work with numbers *and* strings depending on the incoming value type. The pins which accept various types are called *generic*. You can spot them by the little dot in the center.\n\nIf you want to bind a string value to a generic pin, you should enquote it because the number `3.14` and the string `\"3.14\"` are the different things. The quotes explicitly tell XOD you’ve meant a string value.",
```

Comment:
```suggestion
      "content": "## 👆 Generics\n\nNote, the `if-else` node can work with numbers *or* strings depending on the incoming value type. The pins which accept various types are called *generic*. You can spot them by the little dot in the center.\n\nIf you want to bind a string value to a generic pin, you should enquote it because the number `3.14` and the string `\"3.14\"` are different things. The quotes explicitly tell XOD you’ve meant a string value.",
```